### ISSUE TITLE
Test mistake?

### ISSUE THREAD

**Issue by amyjoscelyn** [learn-co-curriculum/OOP-Cards-Model#16]
@markedwardmurray 

In the lab instructions, part III #5, it tells us to add a protection against the deck being empty in the drawNextCard method.

In the tests, however, it says it's checking against the gatherDealtCards method.

**Comment by markedwardmurray** [comment]
yeah, ideally I shouldn't have put that test in that section. I was just reusing the test object that I had made that dealt out the whole deck to check that other method in the rare case that the whole deck had been dealt. It was a bit of an afterthought.

Iceboxing since this is a minor issue, but thanks for pointing it out.
